lib/game.rb

```diff
@@ -0,0 +1,29 @@
+class Game
+  attr_reader :player
+  def initialize(player)
+    @player = player
+  end
+
+  def self.create(player)
+    @game = Game.new(player)
+  end
+  
+  def self.instance
+    @game
+  end
+
+  def choose
+    @game_choice = ["Rock", "Paper", "Scissors"].sample
+  end
+
+  def winner
+    options = ["Rock", "Scissors", "Paper"]
```

Comment:
I like how you have made this method terse here - well done! Another way to model this could be with a hash. An example here: https://github.com/makersacademy/mastery-quiz-exemplars/blob/master/chapter6/quiz1/questions/question_2.rb#L23